plugins/jobtype/src/azkaban/jobtype/HadoopSparkJob.java

```diff
@@ -168,11 +191,34 @@ public void run() throws Exception {
     if (getSysProps().getBoolean(SPARK_DYNAMIC_RES_JOBTYPE_PROPERTY, Boolean.FALSE)) {
       getJobProps().put("env." + SPARK_DYNAMIC_RES_ENV_VAR, Boolean.TRUE.toString());
     }
-    
+
     if (getSysProps().getBoolean(SPARK_NODE_LABELING_JOBTYPE_PROPERTY, Boolean.FALSE)) {
       getJobProps().put("env." + SPARK_NODE_LABELING_ENV_VAR, Boolean.TRUE.toString());
     }
 
+    String sparkPropertyFile = getSysProps().get(SPARK_PROEPRTY_FILE_PATH_JOBTYPE_PROPERTY);
+    File propertyFile = new File(sparkPropertyFile);
+    if (sparkPropertyFile == null || !propertyFile.exists()) {
+      throw new RuntimeException(SPARK_PROEPRTY_FILE_PATH_JOBTYPE_PROPERTY + " must be configured.");
```

Comment:
How would you make sure that the property file is configured properly? If it isn't, would the plugin fail to reload?  Will it prevent AZ executor from starting up?